src/core_plugins/kibana/public/management/sections/objects/components/objects_table/__jest__/__snapshots__/objects_table.test.js.snap

```diff
@@ -3,15 +3,37 @@
 exports[`ObjectsTable delete should show a confirm modal 1`] = `
 <EuiConfirmModal
   buttonColor="primary"
-  cancelButtonText="Cancel"
-  confirmButtonText="Delete"
+  cancelButtonText={
+    <FormattedMessage
+      defaultMessage="Cancel"
+      id="kbn.management.objects.objectsTable.deleteSavedObjectsConfirmModal.cancelButtonLabel"
+      values={Object {}}
```

Comment:
Why is this empty object necessary?

Reply:
It's a snapshot of test. It generates automatically when tests is started.